ke/asm.py

```diff
@@ -152,6 +153,16 @@ def invert(self):
             raise CompileError("Cannot invert %s" % (self,))
         return Boundary(self.reverse, self.character)
 
+"""
+^(?:(?:\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5])\.){3}(?:\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5])$
+^(?:(?:\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5])\.){3}(?:\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5])$
+"""
+class NumberRange(namedtuple("NumberRange", ["start", "end"]), Asm):
+    def to_regex(self, syntax, wrap=False):
+        #force wrap or else faulty output of regex
```

Comment:
1. Are you sure we need to force wrap? Usually our code is good at deciding when it's essential to wrap and when not
2. If we're going to wrap, lets use our `wrap_if()` to do it

Reply:
(you definitely do need to wrap if when`wrap` is `True`)

Reply:
Can't find wrap_if((

Reply:
Called maybe_wrap in python, sorry